x/gov/types/v1/params.go

```diff
@@ -11,21 +11,28 @@ import (
 
 // Default period for deposits & voting
 const (
-	DefaultPeriod                         time.Duration = time.Hour * 24 * 2 // 2 days
+	DefaultPeriod time.Duration = time.Hour * 24 * 2 // 2 days
+	// (New default value for v0.50 migration) 24 hours voting period for expedited proposals.
 	DefaultExpeditedPeriod                time.Duration = time.Hour * 24 * 1 // 1 day
 	DefaultMinExpeditedDepositTokensRatio               = 5
 )
 
 // Default governance params
 var (
-	DefaultMinDepositTokens          = sdkmath.NewInt(10000000)
+	DefaultMinDepositTokens = sdkmath.NewInt(10000000)
+	// During v0.50 migration, this default value is overwritten with existing value of `MinDeposit`.
 	DefaultMinExpeditedDepositTokens = DefaultMinDepositTokens.Mul(sdkmath.NewInt(DefaultMinExpeditedDepositTokensRatio))
 	DefaultQuorum                    = sdkmath.LegacyNewDecWithPrec(334, 3)
 	DefaultThreshold                 = sdkmath.LegacyNewDecWithPrec(5, 1)
-	DefaultExpeditedThreshold        = sdkmath.LegacyNewDecWithPrec(667, 3)
-	DefaultVetoThreshold             = sdkmath.LegacyNewDecWithPrec(334, 3)
-	DefaultMinInitialDepositRatio    = sdkmath.LegacyZeroDec()
-	DefaultProposalCancelRatio       = sdkmath.LegacyMustNewDecFromStr("0.5")
+	// (New default value for v0.50 migration) 75% of Yes votes required for an expedited proposal to pass.
+	DefaultExpeditedThreshold     = sdkmath.LegacyNewDecWithPrec(75, 2)
+	DefaultVetoThreshold          = sdkmath.LegacyNewDecWithPrec(334, 3)
+	DefaultMinInitialDepositRatio = sdkmath.LegacyZeroDec()
+	// (New default value for v0.50 migration) 100% of deposit will not be returned to the depositors,
+	// if the proposal is cancelled. Also, `MsgCancelProposal` is disabled in application.
```

Comment:
Disabling `MsgCancelProposal` will be done in the follow-up PR that points `v4-chain` to this commit.